bridge/processor/span.go

```diff
@@ -180,23 +309,6 @@ func (sp *SpanProcessor) getCurrentChildBlock(ctx context.Context) (uint64, erro
 	return childBlock.Number.Uint64(), nil
 }
 
-// isSpanProposer checks if current user is span proposer
-func (sp *SpanProcessor) isSpanProposer(nextSpanProducers []stakeTypes.Validator) bool {
```

Comment:
So we are completely removing it ? And allow basically every validator to simultaneously submit spans ?